assets/chezmoi.io/docs/user-guide/frequently-asked-questions/troubleshooting.md

```diff
@@ -81,6 +81,24 @@ remove the temporary directory.
     variable to `["--wait"]` or by setting the `EDITOR` environment variable to
     include the `--wait` flag, e.g. `export EDITOR="code --wait"`.
 
+=== "Any editor"
+
+    In this example 'mousepad' is the editor to be used. This program will not wait in the foreground by default and requires DBUS server integration to be disabled (otherwise new PIDs will not be created for the process, if another mousepad instance is already running)
+
+    1. Add a new bash program somewhere in your $PATH (like here: `$HOME/.local/bin/mousepad-wait`)
+    ```bash
+    #!/usr/bin/env bash
+    wait -n; mousepad "$@"
+    ```
+    2. Run `chmod +x $HOME/.local/bin/mousepad-wait` to make the program executable
+    3. Finally add to (or update) `chezmoi.toml`
+    ```toml
+    [edit]
+        command = "mousepad-wait"
+        args = ["--opening-mode=window", "--encoding=UTF-8","--disable-server"]
+    ```
+    or by setting the `EDITOR` environment variable, e.g. `export EDITOR="mousepad-wait --opening-mode=window --encoding=UTF-8 --disable-server"`.
+
```

Comment:
I prefer `shell script` or `bash script` over `bash program`. Given the use of `-n`, this is *absolutely* a `bash script`. It *may* be worth mentioning that `wait -n` will essentially wait for the next child job to complete, if I am understanding this correctly.

There should be blank lines between the code blocks and the paragraphs to which they are attached.

I think that `--opening-mode=window…` should be in the shell-script (which would probably be better named as `chezmoi-mousepad`).

Beyond this, something feels *missing* from this — it’s good text, but it still feels like "mousepad" instructions not "how to make this work with any editor" using "mousepad" as an example.

Reply:
@halostatue, hope you like the changes

Reply:
I think that this is clearer and more easily generalized from this example. I’ll leave it to @twpayne to make the final decision on this one. The only issue I see remaining is the term "any editor" as the title, but I don't have a better suggestion. The Vim and VS Code examples are good examples for any editor with a "foreground" or "wait" command-line parameter, but this one is one for editors with a "we only work in the background" approach.

Reply:
Ok great! Probably the desired length of the tab name restricts more elaborate naming options.